### ISSUE TITLE
fix(starknet): always prove_account on propose

### ISSUE THREAD

**Issue by Sekhmet** [snapshot-labs/sx.js#161]
Currently we won't call `prove_account` if proposer doesn't use singleSlotProof strategy. This means that it might not have been called at all when user tries to vote.

We can either:
1. always call it, no matter what strategies user uses - issues: if someone creates proposal externally it might result in broken proposal, we need to keep track of what strategies are there.
2. call it on vote if we detect it's missing - issues: might slow down voting.

**Comment by bonustrack** [comment]
This is in case someone create a proposal with a space using 2 strategies but use only the non-SSP strategy to verify his voting power? If thats the case we can make so if there is multiple strategies including SSP in the space we force `prove_account`

**Comment by Sekhmet** [comment]
@bonustrack that's right, so that would be solution 1 - one concern is that if that proposal wasn't created by sx.js then people with sx.js won't be able to vote, but I'm not sure if we need to think about it that much.

**Comment by bonustrack** [comment]
@Sekhmet No need to worry about this scenario, extremely unlikely
